packages/edit-site/src/components/site-hub/index.js

```diff
@@ -80,8 +79,7 @@ const SiteHub = memo(
 					<HStack>
 						<div className="edit-site-site-hub__title">
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for site title in site editor mode.

**Note:** It has link variant, hence 40px size does not affect the button.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-title-link-before" src="https://github.com/user-attachments/assets/512b5d30-770d-4c00-86f6-ce45f6c93ec8"> | <img width="419" alt="site-hub-title-link-after" src="https://github.com/user-attachments/assets/94c4dcf9-c9b5-4737-9917-a1d99c36c929"> |